skunkbooth/main.py

```diff
@@ -119,7 +131,21 @@ def CamDimensions(height: int, width: int) -> Tuple[int, int, int]:
                     Scene([PreviewFrame(screen, model=image_selection)], -1, name="Preview")
                 ]
 
-                screen.set_scenes(scenes)
+                screen.set_scenes(scenes, unhandled_input=global_shortcuts)
+
+            if screen.current_scene == scenes[2]:
```

Comment:
We want code specific to a screen to be in the screen's file instead of main.py.

Reply:
Commit 7efeeba should have fixed this. Instead of writing the code directly in main.py, I overrode the process_event method for the Filter Frame class so it would handle the navigational shortcuts required.